bloom-housing/bloom
# 1611/what to expect editable in listings management

# Pull Request Template

## Issue

Addresses #1611


## Description

Changes whatToExpect to be a string instead of a json object. Allows it to be an editable field in listings management but populates it with the default text.

## Type of change


## How Can This Be Tested/Reviewed?

Create a new listing and ensure the default is properly populated and that you can edit it.

## Checklist:
